front50-cassandra/src/main/groovy/com/netflix/spinnaker/front50/model/application/CassandraApplicationDAO.groovy

```diff
@@ -41,7 +41,7 @@ import javax.annotation.PostConstruct
 
 @Slf4j
 @Component
-@ConditionalOnExpression('${spinnaker.cassandra.enabled:false}')
```

Comment:
worth checking the values for `cassandra.enabled` vs `spinnaker.cassandra.enabled` in spinnaker/spinnaker/config